src/Files.App/Helpers/AdaptiveLayoutHelpers.cs

```diff
@@ -3,124 +3,128 @@
 using Files.App.ViewModels;
 using Files.App.ViewModels.Previews;
 using Files.Backend.Services.Settings;
+using IniParser.Model;
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using Windows.Storage;
+using static Files.App.Constants.AdaptiveLayout;
+using IO = System.IO;
 
 namespace Files.App.Helpers
 {
 	public static class AdaptiveLayoutHelpers
 	{
+		private static readonly IFoldersSettingsService foldersSettingsService = Ioc.Default.GetRequiredService<IFoldersSettingsService>();
+
 		public static bool PredictLayoutMode(FolderSettingsViewModel folderSettings, string path, IList<ListedItem> filesAndFolders)
```

Comment:
I find the name of this method a bit missleading. Does it predict what layer mode should be used? Or does it evaluates if the layer mode should be predicted? 
Could use a rename while we're at refactoring it.

Reply:
It's worse because this method modifies the active layout but its name suggests that it only chooses it. I will modify it. The return isn't clear either (true if a new layout was chosen, even if it's the same).

Reply:
The method became void ApplyAdaptativeLayout. The return parameter was never used.